src/page/i-frame.page.ts

```diff
@@ -0,0 +1,15 @@
+import { $, browser, ElementFinder, promise } from 'protractor';
+
+export class IFramePage{
+  private iframe1: ElementFinder;
+
+  constructor () {
+    this.iframe1 = $('#frame1');
+  }
+  public async getIframeHeight(): Promise<number> {
+    return Number((await this.iframe1.getAttribute('height')).slice(0, 3));
```

Comment:
why are you applying slice function?

Reply:
Because I thought when i asked for the attribute it will return 350px not only the number. I didn't check it tho.